content/javascript/concepts/window-prompt/window-prompt.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: 'prompt()'
+Description: 'Displays a dialog with an optional message asking users to input some text.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Parameters'
+  - 'Arguments'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`prompt()`** method is serviced globally by the `window` object provided by the user's browser. It displays a dialog with an optional message asking users to input some text.
+
+## Syntax
+
+`prompt()` expects 1 to 2 parameters. The first one is used to display the title of the dialog. The second is used for an optional default value used in case the user does not input any data, otherwise `prompt()` returns null.
```

Comment:
Pedagogy: would be helpful to include these parameter explanations after the examples of how this function can be composed